modules/layers/src/base-tile-layer/base-tile-layer.js

```diff
@@ -7,14 +7,15 @@ const defaultProps = {
   renderSubLayers: {type: 'function', value: props => new GeoJsonLayer(props), compare: false},
   getTileData: {type: 'function', value: ({x, y, z}) => Promise.resolve(null), compare: false},
   tileToBoundingBox: {type: 'function', value: () => null, compare: false},
-  getTileIndices: {type: 'function', value: () => null, compare: false},
+  getTileIndices: {type: 'function', value: () => [], compare: false},
```

Comment:
Nit; For self-documentation purposes, can we show here what parameters this function takes?